spec/core/matchdata/pre_match_spec.rb

```diff
@@ -0,0 +1,27 @@
+require_relative '../../spec_helper'
+require_relative 'fixtures/classes'
+
+describe "MatchData#pre_match" do
+  # NATFIXME: Implement $` special global variable
+  xit "returns the string before the match, equiv. special var $`" do
+    /(.)(.)(\d+)(\d)/.match("THX1138: The Movie").pre_match.should == 'T'
+    $`.should == 'T'
+  end
```

Comment:
There is a block syntax for `NATFIXME` now, I think it's better to use that whenever possible (since it validates the failure and will croak whenever the issue is resovled):
```ruby
it "returns the string before the match, equiv. special var $`" do
  NATFIXME 'Implement $` special global variable', exception: SpecFailedException do
    /(.)(.)(\d+)(\d)/.match("THX1138: The Movie").pre_match.should == 'T'
    $`.should == 'T'
  end
end
```

See #942 and #963 for more information about this feature

Reply:
Sure, can we update the [code submission policy](https://github.com/natalie-lang/natalie/blob/master/CONTRIBUTING.md#code-submission-policy) if this is now a requirement?

Reply:
I'm not sure if it should be a _requirement_ yet. It's brand new--let's see how it works for a few days before requiring it. I also don't want to put undue burden on newcomers.